### ISSUE TITLE
SSE Engine Fixes (skse64 plugin)

### ISSUE THREAD

**Issue by Dragonisser** [tiltedphoques/Mod-Compatibility#1]
# Mod information

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/17230

#### Dependencies

- SKSE
  -  http://skse.silverlock.org/
  -  link to github mod compatibility issue

# Compatibility

> Mod is not compatible due the nature of its modification to the Skyrim Engine.

# Additional Information

> Most if not all fixes will be natively implemented into Tilted Online

**Comment by FedUpWizard** [comment]
SSE Engine Fixes contained a Fix for False Save Corruption, but Skyrim Together Reborn seemingly doesn't, restarting game will allow you to load the saves, but eventually it will happen again. Check Discord  #Modding for more info. Also check :https://www.reddit.com/r/skyrimmods/comments/8muxmd/sse_engine_fixes_now_with_fix_for_false_save/  and   https://www.reddit.com/r/skyrimmods/comments/ofww46/a_note_on_false_save_corruption_and_your_thoughts/

![image](https://user-images.githubusercontent.com/109249618/178848925-831e9f8b-f80a-4d7a-b3e9-8ba21f641d6a.png)

**Comment by VENDETTA5** [comment]
> SSE Engine Fixes contained a Fix for False Save Corruption, but Skyrim Together Reborn seemingly doesn't, restarting game will allow you to load the saves, but eventually it will happen again. Check Discord #Modding for more info. Also check :https://www.reddit.com/r/skyrimmods/comments/8muxmd/sse_engine_fixes_now_with_fix_for_false_save/ and https://www.reddit.com/r/skyrimmods/comments/ofww46/a_note_on_false_save_corruption_and_your_thoughts/
> 
> ![image](https://user-images.githubusercontent.com/109249618/178848925-831e9f8b-f80a-4d7a-b3e9-8ba21f641d6a.png)

Yeah, also because of that the game does not process dialogue after 10 minutes of gameplay for me and text boxes are empty, can't read books because they are blank and so on.

**Comment by Dragonissa** [comment]
@FedUpWizard @VENDETTA5 https://github.com/tiltedphoques/TiltedEvolution/issues/new?assignees=&labels=Feature&template=Feature_request.md&title=

**Comment by FedUpWizard** [comment]
I have made the feature request Thank you :)

**Comment by Net-Slayer** [comment]
With the feature request by @FedUpWizard completed at Sep 2022:
https://github.com/tiltedphoques/TiltedEvolution/issues/370
Is it now compatible with Skyrim together?

**Comment by Dragonissa** [comment]
> With the feature request by @FedUpWizard completed at Sep 2022: [tiltedphoques/TiltedEvolution#370](https://github.com/tiltedphoques/TiltedEvolution/issues/370) Is it now compatible with Skyrim together?

The issue was only about the save corruption bug and that the fix was natively added by us, instead of hacked in with Engine Fixes. Engine Fixes will **_NEVER_** work with Skyrim Together.

**Comment by Zoups67** [comment]
Where are we at today with the emplementation of SSE Engine Fixes into STR ?

**Comment by miredirex** [comment]
Skyrim Together 1.8 **fully supports** SSE Engine Fixes 7.0.18 and later versions

**Comment by JJuicerr** [comment]
Engine Fixes version 7 is supported. it already  declared on their website
![Image](https://github.com/user-attachments/assets/1230e93b-6814-4839-ac7d-c7a361f4f36e)
